helusers/tests/test_models.py

```diff
@@ -62,3 +66,153 @@ def test_receiving_the_same_logout_token_more_than_once_has_no_effect(self):
         OIDCBackChannelLogoutEvent.objects.logout_token_received(logout_token)
 
         assert OIDCBackChannelLogoutEvent.objects.count() == 1
+
+
+@pytest.mark.django_db
+class TestUserAdGroups:
+    ALL_AD_GROUPS_MAPPING = (
+        ("ad_group_1", "group_1"),
+        ("ad_group_2", "group_2"),
+        ("ad_group_3", "group_3"),
+    )
+    ALL_AD_GROUP_NAMES = ("ad_group_1", "ad_group_2", "ad_group_3")
+    ALL_GROUP_NAMES = ("group_1", "group_2", "group_3")
+
+    @pytest.mark.parametrize(
+        "ad_group_mapping,old_ad_groups_names,old_groups_names,new_ad_groups_names,new_groups_names",
+        [
+            # Nothing changes
+            (
+                ALL_AD_GROUPS_MAPPING,
+                ALL_AD_GROUP_NAMES,
+                ALL_GROUP_NAMES,
+                ALL_AD_GROUP_NAMES,
+                ALL_GROUP_NAMES,
+            ),
```

Comment:
Just FYI, you can do something like this, which would make it more obvious which part of the test fails.
```suggestion
            pytest.param(
                ALL_AD_GROUPS_MAPPING,
                ALL_AD_GROUP_NAMES,
                ALL_GROUP_NAMES,
                ALL_AD_GROUP_NAMES,
                ALL_GROUP_NAMES,
                id="nothing_changes",
            ),
```

Reply:
I added the pytest-params with ids'.